package.json

```diff
@@ -106,6 +106,7 @@
     "react-router-dom": "5.0.1",
     "react-swipe": "6.0.3",
     "react-toolbox": "2.0.0-beta.12",
+    "react-use-promise": "0.2.0",
```

Comment:
Because we use the latest version of React in our project, you can use react `suspense` instead of `react-use-promise. you can find more information on the below link.
https://reactjs.org/docs/concurrent-mode-suspense.html#what-is-suspense-exactly

Reply:
I don't think we should use `Suspense` as the link you posted describes it as experimental and not stable:

> This page describes experimental features that are not yet available in a stable release. Don’t rely on experimental builds of React in production apps